config/initializers/cookies_serializer.rb

```diff
@@ -0,0 +1,5 @@
+# Be sure to restart your server when you modify this file.
+
+# Specify a serializer for the signed and encrypted cookie jars.
+# Valid options are :json, :marshal, and :hybrid.
+Rails.application.config.action_dispatch.cookies_serializer = :marshal
```

Comment:
Was this supposed to be `:hybrid`?

Reply:
No, I need to update the commit message I think instead. I wanted to add this file before the rails-5 change, then we can switch to hybrid later. I better go find the docs / changelog / recommendations again and check what versions have what options available. Please hold!

Reply:
Okay, I found better documentation, and I have updated the documentation to include links to that docs. We can (yay!) use hybrid now.